examples/provider/provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "pipeline" {
+  url = "${var.artifactory_url}"
```

Comment:
Please make no reference to artifactory

Reply:
Modified this.  Although, this is how the same file looks in the `terraform-provider-project` repo.  There are also various references to Artrifactory throughout the provider code and comments as well.  Should these be removed?

Reply:
@alexhung [Bad Dobby!](https://www.youtube.com/watch?v=xCqfwXeq6_8) It shouldn't be that way in projects then. The idea is that there is url, and then we determine the correct endpoint - so, in this case it's for pipelines and has 0 to do with artifactory!

Reply:
Agree! Need to fix in Project provider.